2022-01-12 21:39:25,101 INFO    MainThread:47802 [wandb_setup.py:_flush():71] setting env: {}
2022-01-12 21:39:25,101 INFO    MainThread:47802 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-12 21:39:25,101 INFO    MainThread:47802 [wandb_init.py:_log_setup():371] Logging user logs to /mnt/disk1/vaipe-longnd/OCR-prj-deeplearning/wandb/run-20220112_213925-tvyyla8w/logs/debug.log
2022-01-12 21:39:25,101 INFO    MainThread:47802 [wandb_init.py:_log_setup():372] Logging internal logs to /mnt/disk1/vaipe-longnd/OCR-prj-deeplearning/wandb/run-20220112_213925-tvyyla8w/logs/debug-internal.log
2022-01-12 21:39:25,101 INFO    MainThread:47802 [wandb_init.py:init():404] calling init triggers
2022-01-12 21:39:25,101 INFO    MainThread:47802 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'stride_size': [(2, 2), (2, 2), (2, 1), (2, 1), (1, 1)], 'kernel_size': [(2, 2), (2, 2), (2, 1), (2, 1), (1, 1)], 'pretrained': False, 'emb_dim': 320, 'enc_hid_dim': 216, 'dec_hid_dim': 216, 'dropout': 0.3, 'lr': 0.05, 'epoch': 10, 'batch_size': 32, 'num_workers': 0, 'max_len': 100, 'rnn_type': 'lstm', 'img_channel': 3, 'seed': 1, 'name': 'test'}
2022-01-12 21:39:25,102 INFO    MainThread:47802 [wandb_init.py:init():460] starting backend
2022-01-12 21:39:25,102 INFO    MainThread:47802 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-12 21:39:25,122 INFO    MainThread:47802 [backend.py:ensure_launched():216] starting backend process...
2022-01-12 21:39:25,142 INFO    MainThread:47802 [backend.py:ensure_launched():221] started backend process with pid: 47902
2022-01-12 21:39:25,163 INFO    MainThread:47802 [wandb_init.py:init():469] backend started and connected
2022-01-12 21:39:25,168 INFO    MainThread:47802 [wandb_init.py:init():533] updated telemetry
2022-01-12 21:39:25,197 INFO    MainThread:47802 [wandb_init.py:init():563] communicating current version
2022-01-12 21:39:26,340 INFO    MainThread:47802 [wandb_init.py:init():568] got version response 
2022-01-12 21:39:26,341 INFO    MainThread:47802 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-12 21:39:26,731 INFO    MainThread:47802 [wandb_init.py:init():606] starting run threads in backend
2022-01-12 21:39:27,829 WARNING MainThread:47802 [wandb_init.py:init():952] interrupted
Traceback (most recent call last):
  File "/home/aiotlab/anaconda3/envs/longvaipe/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 931, in init
    run = wi.init()
  File "/home/aiotlab/anaconda3/envs/longvaipe/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 611, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/aiotlab/anaconda3/envs/longvaipe/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 221, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "/home/aiotlab/anaconda3/envs/longvaipe/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 423, in _communicate_run_start
    result = self._communicate(rec)
  File "/home/aiotlab/anaconda3/envs/longvaipe/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 213, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/aiotlab/anaconda3/envs/longvaipe/lib/python3.8/site-packages/wandb/sdk/interface/router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/aiotlab/anaconda3/envs/longvaipe/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/aiotlab/anaconda3/envs/longvaipe/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
